dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -686,6 +686,13 @@ const textRichLink: (format: ArticleFormat) => string = (format) => {
 	return pillarPalette[ArticlePillar.News][400];
 };
 
+const hoverLiveblog = (format: ArticleFormat): string => {
+	if (format) {
+		return pillarPalette[format.theme].main;
+	}
+	return pillarPalette[ArticlePillar.News][400];
```

Comment:
Is news the right default? It might be better to have no default at all and require `theme`